bin/startup.bat

```diff
@@ -65,8 +65,24 @@ set JAVA_OPTS=%JAVA_OPTS% -XX:+UseG1GC
 
 set JAVA_OPTS=%JAVA_OPTS% -Djava.endorsed.dirs=%DOTCMS_HOME%/WEB-INF/endorsed_libs
 
+for /f tokens^=2-5^ delims^=.-_^" %%j in ('java -fullversion 2^>^&1') do set "JAVA_VERSION=%%j%%k"
+echo JAVA_VERSION = %JAVA_VERSION%
```

Comment:
are these echo needed?

Reply:
yeah it would be good to know the version since making decision based on it